cpp/src/arrow/compute/kernels/scalar_temporal_test.cc

```diff
@@ -28,13 +28,25 @@
 #include "arrow/util/checked_cast.h"
 #include "arrow/util/formatting.h"
 #include "arrow/util/logging.h"
+#include "arrow/util/string.h"
 
 namespace arrow {
 
 using internal::StringFormatter;
 
 namespace compute {
 
+template <typename... Elements>
+std::string MakeArray(Elements... elements) {
```

Comment:
Looks this function is not very necessary. Can we directly write the test string at caller side?

Reply:
Done.